required/firstaid/changes.txt

```diff
@@ -1,3 +1,8 @@
+2024-06-25  David Carlisle  <David.Carlisle@latex-project.org>
+
+	* latex2e-first-aid-for-external-files.dtx: add firstaid for arydshln
+	\protect macros to be safe in colortbl
```

Comment:
This `\protect` could be misleading, as the actual firstaid uses `\protected`. Use "Protect macros ..." or "Make macros \protected ..."?